Samples/Notifications/Push/README.md

```diff
@@ -32,12 +35,55 @@ winrt::guid remoteId{ "00000000-0000-0000-0000-000000000000"};
 4. Debug the project. Press **F5**, or select **Debug** \> **Start Debugging**. 
 
 The sample will display a WNS Channel URI which uniquely identify this app, for this user on this device and thus can be used by other apps (usually a server app) to send notifications to this specific app.
+
 ## Receiving Notifications
 Follow these [instructions]() to send a notification to the sample. 
    * Leave the sample running to receive notifications in the foreground.
    * Close the sample to receive notifications in the background.
+
+## Packaged and Unpackaged Apps Considerations
+In addition to making the right API calls, WindowsAppSDK Apps that want to receive Push Notifications must correctly configure their project.
+The type of configuration will depend upon whether the APp is packaged or unpackaged.
+
+### Unpackaged App
+```xml
+<PackagedType>none</PackagedType>
+```
+
+ddlm dll
+
+### Packaged App
+appxmanifest Extension
+
+COM Server
+
+```xml
+<Extensions>
+  <uap:Extension Category="windows.protocol" EntryPoint="Windows.FullTrustApplication">
+    <uap:Protocol Name="windowsappsdktestprotocol-packaged">
+      <uap:DisplayName>Windows App SDK TestProtocol</uap:DisplayName>
+    </uap:Protocol>
+  </uap:Extension>
+  <com:Extension Category="windows.comServer">
+    <com:ComServer>
+      <com:ExeServer Executable="PushNotificationsTestApp\PushNotificationsTestApp.exe" DisplayName="SampleBackgroundApp" Arguments="----WindowsAppSDKPushServer">
+        <com:Class Id="[Your app's Azure AppId]" DisplayName="Windows App SDK Push" />
+      </com:ExeServer>
+    </com:ComServer>
+  </com:Extension>
+</Extensions>    
+```
+
+Take notice that the Packaged sample has a special extension to the COM server and protocol used to deliver the notification to the app.
+
+---
+**How is an upackaged sample get the notification?**
+
+*If the packaged sample requires the extensions for the COM server and protocol in order to receive notifications, how does the unpackage sample manages to receive notification has it does not have an appxmanifest?*
```

Comment:
nit: *how does the "unpackaged" sample manage to receive notification *as* it does not have an appxmanifest file?*

Reply:
thanks